disco_aws_automation/disco_aws.py

```diff
@@ -422,7 +422,7 @@ def provision(self, ami, hostclass=None,
             group_name=group_name
         )
 
-        self.create_scaling_schedule(hostclass, min_size, desired_size, max_size)
+        self.create_scaling_schedule(min_size, desired_size, max_size, group_name=group.name)
```

Comment:
it'd also be nice to be a bit more explicit and call it `asg` or something comparable... `group` seems too generic to me